src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -105,40 +105,81 @@ public function remove(string $key)
      */
     public function getAlpha(string $key, string $default = ''): string
     {
-        return preg_replace('/[^[:alpha:]]/', '', $this->get($key, $default));
+        return preg_replace('/[^[:alpha:]]/', '', $this->getString($key, $default));
```

Comment:
Currenty, there is a type error if the value is an array.
We can do a bugfix to revert the return type definition and allow `string|array`.